### PR TITLE
simple-freedom-chat cleanup

### PR DESCRIPTION
**Pull request by trevj**
[Review on Reviewable](https://reviewable.io/reviews/uproxy/uproxy-lib/207)

### CODE REVIEW

File: src/samples/simple-freedom-chat/freedom-module.ts [hunk 1]
```diff
@@ -3,106 +3,92 @@
...
+  channel.onceOpened.then(() => {
+    log.info('%1: channel opened', name);
+  });
+  channel.onceClosed.then(() => {
+    log.info('%1: channel closed', name);
+  });
 
-  parentFreedomModule.on('send' + name, (message:Message) => {
-    d.send({str: message.message})
+  // Forward messages from the UI to the remote peer.
```
Comment (bemasc) [inline src/samples/simple-freedom-chat/freedom-module.ts]: Are these comments backward?

File: src/samples/simple-freedom-chat/freedom-module.ts [hunk 2]
```diff
@@ -3,106 +3,92 @@
...
-});
-b.signalForPeerQueue.setSyncHandler((message:signals.Message) => {
-  log.info('b: sending signalling message to a.');
-  a.handleSignalMessage(message);
-});
+a.signalForPeerQueue.setSyncHandler(b.handleSignalMessage);
+b.signalForPeerQueue.setSyncHandler(a.handleSignalMessage);
 
-// Negotiate a peerconnection. Once negotiated, enable the UI and add
+// Have a negotiate a connection. Once negotiated, enable the UI and add
```
Comment (bemasc) [inline src/samples/simple-freedom-chat/freedom-module.ts]: Grammar

**Comment by bemasc** [comment]
:+1:

File: src/samples/simple-freedom-chat/freedom-module.ts (hunk 1 above)
Comment (trevj) [inline src/samples/simple-freedom-chat/freedom-module.ts]: Yes! Thanks. Fixed.

File: src/samples/simple-freedom-chat/freedom-module.ts (hunk 2 above)
Comment (trevj) [inline src/samples/simple-freedom-chat/freedom-module.ts]: Sure. I still don't really know where that convention came from but done.

**Pull request merged**